templates/addons/cilium-network-policies/ciliumNetworkPolicies.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CLUSTER_NAME}-cilium-policy
+data:
+  cilium-policy.yaml: |-
+    apiVersion: "cilium.io/v2"
+    kind: CiliumClusterwideNetworkPolicy
+    metadata:
+      name: "default-cluster-policy"
+    spec:
+      description: "allow cluster intra cluster traffic"
+      endpointSelector: {}
+      ingress:
+        - fromEntities:
+            - cluster
+        - fromCIDR:
+            - 10.0.0.0/8
+    ---
+    apiVersion: "cilium.io/v2"
+    kind: CiliumClusterwideNetworkPolicy
+    metadata:
+      name: "default-external-policy"
+    spec:
+      description: "allow etcd & api server traffic"
+      nodeSelector: {}
+      ingress:
+        - fromEntities:
+            - world
+          toPorts:
+            - ports:
+                - port: "6443"
+                - port: "2379"
+                - port: "2380"
+---
+apiVersion: addons.cluster.x-k8s.io/v1beta1
+kind: ClusterResourceSet
+metadata:
+  name: ${CLUSTER_NAME}-cilium-policy
+spec:
+  clusterSelector:
+    matchLabels:
+      cluster: ${CLUSTER_NAME}
+  resources:
+    - kind: ConfigMap
+      name: ${CLUSTER_NAME}-cilium-policy
+  strategy: ApplyOnce
```

Comment:
Is there a reason for using `ApplyOnce`? My understanding is that in case this fails, controller will never try again. Maybe it should be `Reconcile`?

Reply:
I believe based on the [CAEP](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/proposals/20200220-cluster-resource-set.md) it will only successfully apply once and not check for drift after that. I think it is safe to switch this to reconcile though

Reply:
I've updated the PR to change these all to reconcile